ts/components/screens/BaseHeader.tsx

```diff
@@ -229,10 +234,10 @@ class BaseHeaderComponent extends React.PureComponent<Props, State> {
               <IconFont name={customRightIcon.iconName} />
             </ButtonDefaultOpacity>
           )}
-        {fromNullable(this.props.accessibilityEvents).fold(
-          true,
-          ({ avoidNavigationEventsUsage }) => !avoidNavigationEventsUsage
-        ) && <NavigationEvents onDidFocus={this.handleFocus} />}
+        {!onShowHelp &&
+          !customRightIcon &&
+          showInstabugChat === false &&
```

Comment:
The miss-alignment still remains.

<img src="https://user-images.githubusercontent.com/822471/91739031-a2b91a00-ebb1-11ea-9630-e5cdacc4bc46.png" height="600"/>


maybe this could fix
```suggestion
          !showInstabugChat &&
```

Reply:
@Undermaken yes, it still remains even applying your proposed change... I'm actually working on a possible fix but it looks like the combination of `Left`, `Body` and `Right` UI components keeps the behaviour